libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxNavigationTest.kt

```diff
@@ -212,6 +218,100 @@ class MapboxNavigationTest {
         verify(exactly = 0) { fasterRouteObserver.onFasterRouteAvailable(routes[0]) }
     }
 
+    @Test
+    fun reRoute_called() {
+        val offRouteObserverSlot = slot<OffRouteObserver>()
+        verify { tripSession.registerOffRouteObserver(capture(offRouteObserverSlot)) }
+
+        offRouteObserverSlot.captured.onOffRouteStateChanged(true)
+
+        val optionsSlot = slot<RouteOptions>()
+        verify(exactly = 1) { directionsSession.requestRoutes(capture(optionsSlot), any()) }
+    }
+
+    @Test
+    fun reRoute_called_with_null_bearings() {
```

Comment:
Minor thing.

Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert
If the `A`rrange part gets too long we should create factory or builder methods.

Reply:
fixed